m2cgen/interpreters/javascript/utils.js

```diff
@@ -0,0 +1,15 @@
+function dict_null_to_nan(input) {
+    for (const [key, value] of Object.entries(input)) {
+        if (value == null)
+            input[key] = NaN;
+    }
+    return input;
+}
+
+function dict_to_value_array(input) {
+    var output = [];
+    for (const elem in input) {
+        output.push(input[elem]);
+    }
+    return output;
+}
```

Comment:
What is the purpose of these functions? They do not seem to be used by `JavascriptCodeGenerator`.

Reply:
I second this. What's the idea behind the `with_util_functions` parameter?

Reply:
The util functions are mostly used within BigQuery.  I was on the fence about adding them in as they aren't used by M2CGEN itself but they make passing inputs in much easier.  BigQuery has a TO_JSON_STRING() function that takes a row as input making this very easy to pass JSON to a JavaScript function.  Once it's in JavaScript we always need to convert `null` to `NaN` and if we choose to use the default array input we need to convert the dict to an array.  We can do the conversion ahead of time but it's much easier with the util functions. Also, the way we're using the JavaScript models is first converting the code then string search/replacing `input[0]` to `input.featureA`.

Reply:
If you'd like me to exclude these and the corresponding parameter let me know.

Reply:
@bcampbell-prosper I'd say these functions are quite specific to the BigQuery use case and are outside of the `m2cgen`'s domain. This is why I suggest to remove them. 

I believe it shouldn't be too hard to include them in your own build process right after the generated code is produced. You can, for instance, append those functions to the generated code string.

Reply:
Makes sense.  I'll take them out.